app/src/main/contrib/components/ChooseGroup/ChooseGroupContent.tsx

```diff
@@ -1,59 +1,56 @@
 import * as React from "react";
+import { connect } from 'react-redux';
+
 import { ModellingGroup } from "../../../shared/models/Generated";
-import { responsibilityStore } from "../../stores/ResponsibilityStore";
-import { contribAuthStore } from "../../stores/ContribAuthStore";
-import { connectToStores } from "../../../shared/alt";
 import { GroupList } from "./GroupList";
-import { RemoteContentComponent } from "../../../shared/components/RemoteContentComponent/RemoteContentComponent";
-import { RemoteContent } from "../../../shared/models/RemoteContent";
-import { InternalLink } from "../../../shared/components/InternalLink";
 import { ButtonLink } from "../../../shared/components/ButtonLink";
+import { modellingGroupsActions } from "../../actions/modellingGroupsActions";
+import { LoadingElement } from "../../../shared/partials/LoadingElement/LoadingElement";
+// import { GlobalState } from "../../reducers/reducers";
 
 import "../../../shared/styles/common.scss";
 
-export interface ChooseGroupProps extends RemoteContent {
+export interface ChooseGroupProps {
     groups: ModellingGroup[];
+    dispatch: any;
```

Comment:
I do think it's nice to hide `dispatch` away, rather than expose it as a property. Could we wrap up the call to get groups into a function prop, and use a `mapDispatchToProps` connector? Open to being convinced otherwise if you think there's a good reason to leave it as is! But in that case lets add a type to `dispatch`.